docs/tools/vdb_table/data/cratedb.json

```
@@ -5,7 +5,7 @@
     "github": "https://github.com/crate/crate",
     "website": "http://www.cratedb.com/",
```

Not using https, please adjust. Thanks!
```suggestion
    "website": "https://cratedb.com/",
```

---

thx